### DISCUSSION TITLE
[Feature]: Show album owner

### DISCUSSION THREAD

**Discussion by oddlama** [immich-app/immich#1656] [Feature Request]
### Feature detail

Just a minor thing: When I currently share an album with a person, only the shared persons are shown in the list of people at the top. This is a little confusing for the other users, it appears as if the owner of the album isn't actually added to the album.

![image](https://user-images.githubusercontent.com/31919558/192112759-a32758c3-70c7-42c5-97a4-30a6602166aa.png)

Two people have access to this album, but it looks like only one person does.

### Platform

Web

**Comment by Spyy004** [comment]
Hey, can you assign me this? I am working on it

**Comment by Spyy004** [comment]
can you please tell me the file in the directory which needs inspection?

**Comment by bo0tzz** [comment]
@Spyy004 The rendering for the web client's album members is at https://github.com/immich-app/immich/blob/main/web/src/lib/components/album-page/album-viewer.svelte#L420-L438. For the mobile app, it's at https://github.com/immich-app/immich/blob/main/mobile/lib/modules/album/views/album_viewer_page.dart#L158-L184.

**Comment by bo0tzz** [comment]
@Spyy004 are you still working on this, or should I unassign you?

**Comment by martyfuhry** [comment]
If you update the `AlbumResponseDTO` (or maybe it's now becoming `Album`) to replace`ownerId` with a `OwnerResponseDTO owner`, then I can implement this very easily.

**Comment by jrasm91** [comment]
We can probably make a change in the api to support this, but I have a feeling owner id isn't currently a foreign key, so it'll take a little bit more effort to do it 😛

**Comment by bo0tzz** [comment]
I believe this is implemented now
